packages/store-web/src/dexie/dexie.ts

```diff
@@ -51,72 +51,5 @@ export class DexieDatabase extends Dexie {
     this.entities = this.table(DexieTable.Entities);
     this.derived_taskComponents = this.table(DexieTable.DerivedTaskComponents);
     this.metadata = this.table(DexieTable.Metadata);
-
-    this.entities.hook("creating", onEntitiesInsert);
-    this.entities.hook("updating", onEntitiesUpdate);
-    this.entities.hook("deleting", onEntitiesDelete);
-  }
-}
-
-function onEntitiesInsert(
-  primaryKey: number,
-  obj: DexieEntityRow,
-  transaction: Transaction,
-) {
-  insertDerivedTaskComponent(obj, transaction);
-}
-
-async function onEntitiesUpdate(
-  // `Object` type is needed to conform to Dexie types
-  // eslint-disable-next-line @typescript-eslint/ban-types
-  modifications: Object,
-  primaryKey: number,
-  obj: DexieEntityRow,
-  transaction: Transaction,
-) {
-  if (obj.entityType === EntityType.Task) {
-    await deleteAllDerivedTaskComponentsByTaskID(obj.id as TaskID, transaction);
-
-    insertDerivedTaskComponent({ ...obj, ...modifications }, transaction);
   }
 }
-
-async function onEntitiesDelete(
```

Comment:
n.b. there's no equivalent in my patch; entities can never be deleted once created (would violate our event sourcing semantics)—instead they can have an isDeleted flag or whatever